src/test/java/unit/tests/utils/MyDateTests.java

```diff
@@ -0,0 +1,161 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+
+package unit.tests.utils;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import core.utils.MyDate;
+

```

Comment:
this line should be deleted

Reply:
The IntelliJ formatter fixes it like that, probably a difference in conventions.

Reply:
Done.